packages/react/src/PasswordInput/PasswordInput.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef } from 'react'
+import type { ForwardedRef, InputHTMLAttributes } from 'react'
+
+export type PasswordInputProps = {
+  /** Whether the value fails a validation rule. */
+  invalid?: boolean
+} & Omit<InputHTMLAttributes<HTMLInputElement>, 'aria-invalid' | 'type'>
+
+export const PasswordInput = forwardRef(
+  ({ className, invalid, ...restProps }: PasswordInputProps, ref: ForwardedRef<HTMLInputElement>) => (
+    <input
+      {...restProps}
+      aria-invalid={invalid || undefined}
+      className={clsx('ams-password-input', className)}
+      ref={ref}
```

Comment:
Did you remove `dir={dir ?? 'auto'}` here on purpose? A password can be in a RTL language as well, no?

Reply:
No not on purpose. It’s in now.